README.md

```diff
@@ -1 +1,88 @@
-# angular-concepts-readme
+# Concepts
```

Comment:
Give this a more descriptive title - 'Angular Concepts' or 'Angular Core Concepts' would be better